docs/installation/README.md

```diff
@@ -1,309 +1,245 @@
-
-You can find more information about creating/adding SSH keys in [this article](https://www.digitalocean.com/docs/droplets/how-to/add-ssh-keys/).
-
-![](/images/addsshkey.png)
-
-4. You will then be able to start a CloudLinux OS Droplet using the image.
-   
-   :::tip Note
-   Your Droplet will be created in the same datacenter that your custom image resides in.
-   :::
 
-![](/images/startdroplet.png)
-
-5. Now, use your preferred SSH client software to connect to your Droplet. You can find more information on SSH client setup [here](https://www.digitalocean.com/docs/droplets/how-to/connect-with-ssh/).
+The script automatically detects and supports the following control panels:
+* cPanel with Easy Apache 4
 
-![](/images/sshclient.png)
+Unfortunately, for now is is not available to convert servers with:
+* Plesk
+* DirectAdmin
+* Other control panels that use [CloudLinux OS Shared integration](https://docs.cloudlinux.com/control_panel_integration/#introduction).
 
-## Uninstalling
+Please subscribe to [our blog](https://blog.cloudlinux.com/), check news and receive updates. 
 
-You can always uninstall CloudLinux OS. In this case, the system will be converted back to CentOS (even if the original system was RHEL)
 
-The following actions will be taken:
+:::tip cldeploy explained
 
-1. CloudLinux repositories & <span class="notranslate">yum</span> plugin will be removed.
-2. CentOS repositories will be set up.
+By its design, CloudLinux OS Solo is very close to the upstream operating system, RHEL (and so, CentOS and AlmaLinux). 
+This makes the conversion process relatively straightforward, requiring just one reboot. 
 
-In the end, the script will provide instructions on how to finish the conversion back to CentOS. 
+Here's what the cldeploy script does when you run it:
 
-:::warning
-Do not forget to free up a CloudLinux OS license by removing the server from the [Servers section of your CLN account](https://docs.cln.cloudlinux.com/dashboard/#servers). After that, if you don't intend to use the license anymore, you can [remove it](https://docs.cln.cloudlinux.com/dashboard/#cloudlinux-os-activation-keys) to avoid being billed for it. 
+* Backups the original repository settings into <span class="notranslate">`/etc/cl-convert-saved`</span>.
+* Backups RHEL system ID into <span class="notranslate">`/etc/cl-convert-saved`</span> (RHEL systems only).
+* Installs CloudLinux repository settings and imports CloudLinux RPM key.
+* Replaces redhat/centos-release, redhat-release-notes, redhat-logos with CL version.
+* Re-installs CL version of rhnlib/rhnplugin.
+* Checks for binary kernel modules, finds replacement if needed.
+* Detects OVH servers and fixes mkinitrd issues.
+* Detects Linode servers and fixes grub issues.
+* Checks if LES is installed.
+* Checks that <span class="notranslate">`/etc/fstab`</span> has correct <span class="notranslate">`/dev/root`</span>
+* Checks for efi.
+* Installs CloudLinux Manager for cPanel Solo.
```

Comment:
why only cPanel Solo? I thought we support regular cPanel as well

Reply:
We don't support resellers and block UI when more that a single user exists on server, so regular cPanel support is rather 'bug', not 'feature'.